js/main.js

```diff
@@ -308,7 +308,15 @@ Featured = {
     var _this = this;
 
     if ( $('#featured-post-container').length ) {
+
+      // Get image height
+      $maxHeight = $('#featured-post-image-holder img').height();
       
+      // Set max heights
+      if( $(window).width() > 720 ) {
+        $('#featured-post-container, #featured-post-title-holder, #featured-post-subtitle-holder, #featured-post-subtitle, #featured-post-image-holder').css('max-height', $maxHeight + 'px');
+      }
```

Comment:
needs an else to explicitly set height to `auto` if the window is resized from e.g. 780 to 650